controller/array_action/array_mediator_ds8k.py

```diff
@@ -409,24 +405,20 @@ def _get_pools(self):
         logger.info("Pools found: {}".format(pools))
         return pools
 
-    def _get_api_volume_by_name(self, volume_name, not_exist_err=True):
+    def _get_api_volume_by_name(self, volume_name):
         logger.info("Getting volume {}".format(volume_name))
         pools = self._get_pools()
         volume_candidates = []
 
         for pool in pools:
```

Comment:
fetching all volumes is very heavy to performance. This problem exists fro m before in get_volume
Opened relevant ticket: CSI-1511

Reply:
done

Reply:
CSI-1511 description is about fetching all volumes in *specific* pool, but this code line shows that all pools are "scanned", which I think [can be avoided](https://github.com/IBM/ibm-block-csi-driver/pull/163#discussion_r433846254)